### DISCUSSION TITLE
Server Requirements to Monitor 100 websites

### DISCUSSION THREAD

**Discussion by muhammadhanif** [umami-software/umami#1855] [Q&A]
I will be using Umami to monitor visitor traffic on the 100 websites that I maintain. 

I'd would like to ask your advice on the server requirements that I have to prepare: the amount of RAM, vCPU, storage, and etc. I plan to use Ubuntu Server 22.04 (Virtual Machine).

Thank you, and I look forward to hearing from you.

**Comment by franciscao633** [comment]
Are you going with MySQL or Postgres and how many monthly hits are you expecting across the 100 websites? The database will have a noticeable performance hit when trying to aggregate > 5 million pageviews in a single query, but if its low traffic, its possible. Our cloud version should be able to handle the traffic, but you need to weigh the options between self-hosting and cloud. I think it will be highly dependent on your volume.
